src/create-index.ts

```diff
@@ -69,18 +69,20 @@ export type IndexType = {
     keyItemGroupMap: Map<Midashi, ItemGroup[]>;
     SudachiSynonymsItemGroup: Map<SudachiSynonyms, ItemGroup>;
 };
-let _ret: IndexType | null = null;
-export const createIndex = async (): Promise<IndexType> => {
-    if (_ret) {
-        return Promise.resolve(_ret);
+const _cache = new Map<boolean, IndexType>();
+const fixedVocabularyNumber = 1;
```

Comment:
variable name should describe why the value is `1`.
I think that it would be firstVocabularyNumber.